sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/ChatThreadClient.java

```diff
@@ -298,17 +298,17 @@ public ChatMessage getMessage(String chatMessageId) {
     /**
      * Gets the list of thread messages.
      *
-     * @param listChatMessagesOptions the list options.
+     * @param listMessagesOptions the list options.
      * @param context the context to associate with this operation.
      *
      * @return the list of thread messages.
      */
     @ServiceMethod(returns = ReturnType.COLLECTION)
-    public PagedIterable<ChatMessage> listMessages(ListChatMessagesOptions listChatMessagesOptions,
+    public PagedIterable<ChatMessage> listMessages(ListChatMessagesOptions listMessagesOptions,
```

Comment:
Should we rename the type to `ListMessagesOptions` as well?

Reply:
Java also uses type name ListChatMessagesOptions. 

https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/communication/azure-communication-chat/src/main/java/com/azure/communication/chat/ChatThreadAsyncClient.java#L531

Reply:
Usually, as Victor pointed, the option bag name follows the pattern of `<simple-method-name>Options`, but it seems, java took the route of having "Chat" text in the Options model name e.g. "AddChatParticipantsOptions" not "AddParticipantsOptions", it could be that java used more specialized names to future proof, like in future if there is another client (not-chat) in the same module where it has API to add participants Or the case where Chat used with Call sdk which may support adding participants to a call.